### PR TITLE
Add Python 3.12 in tests and other lists

### PR DESCRIPTION
**Pull request by markkuleinio**
### Fixes: #600

Added Python 3.12 in tests, setup.py classifiers and in the bug report forms.

### CODE REVIEW

**Comment by markkuleinio** [comment]
I did not change `.readthedocs.yaml` yet, I guest that's just for building the docs, maybe that can be done later.

**Comment by markkuleinio** [comment]
I don't know how to fix the failing tests, @abhi1693 

>  Container pytest_pynetbox_1704357384-netbox_v3_3_netbox-housekeeping-1  Created
Error response from daemon: failed to mkdir /var/lib/docker/volumes/pytest_pynetbox_1704357384_pytest_pynetbox_v3_3_netbox-media-files/_data/image-attachments: mkdir /var/lib/docker/volumes/pytest_pynetbox_1704357384_pytest_pynetbox_v3_3_netbox-media-files/_data/image-attachments: file exists

**Comment by abhi1693** [comment]
It's possible that 3.12 was not supported with older netbox versions.

**Comment by markkuleinio** [comment]
> It's possible that 3.12 was not supported with older netbox versions.

Why would that be shown as a docker volume creation error? (Sorry, I'm not a dockerist)

**Comment by markkuleinio** [comment]
I'll still try with NetBox 3.7 only, just a minute

**Comment by markkuleinio** [comment]
I don't see the logic: how it failed the Python 3.8 + NetBox 3.3 test. Something is wrong in the docker logic, cannot help that. Restoring the original changes now

**Comment by markkuleinio** [comment]
Looks like the docker image for NetBox 3.3 is not working, others are fine.

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -30,6 +30,8 @@ body:
         - "3.9"
         - "3.10"
         - "3.11"
+        - "3.12"
+        - "3.13alpha/beta"
```
Comment (abhi1693) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:34]: I don't think we should test on 3.13 until it's stable release

**Review by abhi1693 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yaml (hunk 1 above)
Comment (markkuleinio) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:34]: It's not the test list, it's the bug report template dropdown menu, making it possible for submitting a bug report when running any currently available Python version:

image

Comment (abhi1693) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:34]: We will add it when we are running tests for it. I don't want to give an illusion to the end user that this version is also supported

**Review by abhi1693 — APPROVED** [review]

**Pull request closed without merging**